user_docs/en/changes.md

```diff
@@ -43,6 +43,7 @@ The available options are:
   * It is now possible to use braille display routing keys to move the text cursor. (#9101)
   * It is now possible to use the review cursor selection commands to select text.
 * Updating NVDA while add-on updates are pending no longer results in the add-on being removed. (#16837)
+* NVDA is no longer as sluggish when arrowing up and down through large files in VS Code. (#17039)
```

Comment:
Does it mean that it remains sluggish anyway, but less than before this PR?

If not, maybe the following would be more suitable?
```suggestion
* NVDA is no longer  sluggish when arrowing up and down through large files in VS Code. (#17039)
```

Reply:
I think performance could still be better. This at least reduces the slowdown seen in 2024.3.